package.json

```diff
@@ -34,12 +34,17 @@
     "watch:build": "onchange 'index.js' --initial -- npm run build -- --continue-on-error",
     "watch": "npm-run-all --parallel --silent watch:*",
     "prepublish": "npm run build",
-    "test": "xo; npm run build"
+    "test": "xo && jest; npm run build"
   },
   "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.2.3",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
There are two babel-core

Reply:
All of these packages are referenced in the [Jest readme](https://jestjs.io/docs/en/getting-started.html#using-babel)

Reply:
That's just sad

Reply:
Maybe it's not necessary if babel < 7 is used. Let me see.

Reply:
I'd wait for this 

https://github.com/facebook/jest/issues/4842#issuecomment-452017187